locations/spiders/audi.py

```diff
@@ -0,0 +1,55 @@
+import requests
+from scrapy import Spider
+from scrapy.http import JsonRequest
+
+from locations.items import Feature
+
+
+class AudiSpider(Spider):
+    name = "audi"
+    item_attributes = {
+        "brand": "Audi",
+        "brand_wikidata": "Q23317",
+    }
+    graphql_url = "https://dev-dealer-graphql.apps.emea.vwapps.io/"
+
+    def start_requests(self):
+        market_request = requests.post(
```

Comment:
I'd vote against `requests` being used here, I'd yield a `JsonRequest`, let scrapy do it's thing then another callback to yield the market requests.

Reply:
updated :-)